README.md

```diff
@@ -8,25 +8,20 @@
 - [Release Status](#status)
 - [Join MainNet](#mainnet)
 - [Join Mainnet without using Docker](#mainnet_native)
-- [Join TestNet](#testnet)
 - [Interact with Blockchain](#ioctl)
 - [Enable Logrotate](#log)
 - [Operate Your Node](#ops)
 - [Upgrade Your Node（One Line Upgrader）](#upgrade)
 - [Q&A](#qa)
 
-
 ## <a name="status"/>Release Status
 
 Here are the software versions we use:
 
-- IoTeX Mainnet: v1.10.1
-- IoTeX Testnet: v1.10.1
+- IoTeX MainNet: v1.10.1
 
-<!--
-- MainNet: v1.9.2
-- TestNet: v1.9.2
--->
+**Note**
+To start and run a testnet node, please click [**Join Testnet**](https://github.com/iotexproject/iotex-bootstrap/blob/master/README_testnet.md)
```

Comment:
## <a name="testnet"/>Join TestNet
To start and run a testnet node, please click [**Join Testnet**](https://github.com/iotexproject/iotex-bootstrap/blob/master/README_testnet.md)